dev/tests/integration/testsuite/Magento/Framework/View/Model/Layout/MergeTest.php

```diff
@@ -77,10 +130,24 @@ protected function setUp(): void
         $this->model = $objectManager->create(
             \Magento\Framework\View\Model\Layout\Merge::class,
             [
+                'design' => $design,
+                'scope' => $scopeResolver,
+                'fileSource' => $fileSource,
+                'pageLayoutFileSource' => $pageLayoutFileSource,
+                'appState' => $this->appState,
+                'cache' => $this->cache,
+                'layoutValidator' => $this->layoutValidator,
+                'logger' => $this->logger,
+                'readFactory' => $readFactory,
                 'theme' => $theme,
+                'cacheSuffix' => '',
                 'layoutCacheKey' => $this->layoutCacheKeyMock,
+                'serializer' => $this->serializer,
+                'cacheLifetime' => self::STUB_DEFAULT_CACHE_LIFETIME
             ]
         );
+
+
     }
 
```

Comment:
Do we need pass all arguments from constructor in test ? Found some other place only have fewer arguments

Reply:
Basically, you don't need to pass all the arguments in the integration test. You need to pass only mocks if you have any. Suppose the original class is OK for you - no need to pass it to the constructor.